### PR TITLE
[terra-clinical-data-grid] Decorative row error

### PR DESCRIPTION
**Pull request by benbcai**
### Summary

- This PR also publishes the fix so I don't need to issue a separate release PR.
- Where all rows in all sections are marked as decorative (`isDecorative`), the following error is thrown in `getFirstAndLastVisibleRowData` because no decorative rows are found to access the id.

Screen Shot 2022-04-20 at 1 38 52 PM

Closes: UXPLATFORM-6501

### Deployment Link

https://terra-clinical-deployed-pr-#.herokuapp.com/

### Testing
This change has been tested by providing a packed .tgz file to the engineer who reported the issue and verified that the error no longer occurs with this change.

### Additional Details

Thank you for contributing to Terra.
@cerner/terra

### CODE REVIEW

File: packages/terra-clinical-data-grid/package.json
Comment (sdadn) [inline packages/terra-clinical-data-grid/package.json:4]: Doesn't this get updated during the release process?

Comment (benbcai) [reply packages/terra-clinical-data-grid/package.json:4]: @sdadn Yes, I indicated in the PR description that this PR includes the version bump for publishing because I wanted to get this released ASAP and not wait until for week to unblock that team needing this. That way I don't have to issue a separate PR, wait for it build and get approvals again.

**Review by BenBoersma — APPROVED** [review]

**Review by sdadn — APPROVED** [review]

**Pull request merged**
